dfs/registry/pull.go

```diff
@@ -32,7 +32,7 @@ var (
 // Registry performs specific docker actions based on the registry index
 type Registry interface {
 	SetConnection(conn client.Connection)
-	PullImage(cancel <-chan time.Time, image string) error
+	PullImage(cancel <-chan time.Time, image string) (string, error)
```

Comment:
Why do we need to return the string of the image that we are pulling? Is the image field not sufficient?